kolibri/core/assets/src/constants.js

```diff
@@ -27,6 +27,7 @@ export const ContentNodeKinds = {
   CLASSROOM: 'CLASSROOM',
   ACTIVITY: 'ACTIVITY',
   SLIDESHOW: 'slideshow',
+  BOOKMARK: 'bookmark',
```

Comment:
Bookmarks are not ContentNodes, but are only references to ContentNode so this should be removed. I don't see anywhere that you're using `ContentNodeKinds.BOOKMARK` so removal shouldn't impact anything else unless I'm missing the usage.

Reply:
I added this line because when I pass the prop kind in the `LessonContentCard `component in `CreateExamPage/index.vue` file in one case the `kind `is bookmark. in that case the received prop goes through validation
![Screenshot 2021-08-24 180409](https://user-images.githubusercontent.com/52789571/130641154-e8d313e2-6647-44dd-bcbe-268753bbe67a.png)
and that function goest to see if the string bookmarks is part of the `ContentNodeKinds` object. 
![Screenshot 2021-08-24 180743](https://user-images.githubusercontent.com/52789571/130641761-22cd7b90-ad49-4a26-a116-b1f53205aaed.png)
If i remove this line, i receive an error that check failed for kind
![Screenshot 2021-08-24 180902](https://user-images.githubusercontent.com/52789571/130642022-1aeedef3-2b39-419c-9b32-606fe30a45eb.png)
I think one option is to rework the `ValidateContentKind` function, but I might break something else